content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -65,25 +61,13 @@ DeviceClass
   when installing a resource driver. Each request to allocate a device
   in a ResourceClaim must reference exactly one DeviceClass.
 
-PodSchedulingContext
-: Used internally by the control plane and resource drivers
-  to coordinate pod scheduling when ResourceClaims need to be allocated
-  for a Pod and those ResourceClaims use a control plane controller.
-
 ResourceSlice
-: Used with structured parameters to publish information about resources
+: Used by DRA drivers to publish information about resources
   that are available in the cluster.
 
-The developer of a resource driver decides whether they want to handle
-allocation themselves with a control plane controller or instead rely on allocation
-through Kubernetes with structured parameters. A
-custom controller provides more flexibility, but cluster autoscaling is not
-going to work reliably for node-local resources. Structured parameters enable
-cluster autoscaling, but might not satisfy all use-cases.
-
-When a driver uses structured parameters, all parameters that select devices
-are defined in the ResourceClaim and DeviceClass with in-tree types. Configuration
-parameters can be embedded there as arbitrary JSON objects.
+All parameters that select devices are defined in the ResourceClaim and
+DeviceClass with in-tree types. Configuration parameters can be embedded there
+as arbitrary JSON objects.
```

Comment:
_Aside (not linked to classic DRA removal)_

Do they have to be JSON? I think we're describing an implementation detail.

If you use the API through YAML, or even soon CBOR, can you still provide an arbitrary structure here.

Reply:
They have to be embedded as JSON. API validation checks that.

Reply:
> the value of the field is a string in JSON format, even if the client sends YAML or CBOR as the request body?

After decoding of what the client sent, the raw bytes must be valid JSON. So if someone sends with protobuf encoding, the string must be JSON.

Reply:
OK. Best to follow up on another PR. Good to clarify this for beta though.

Reply:
As I'm touching it here and we started this discussion, let's fix it also in this PR. How about this?

> Configuration parameters can be embedded there.
> What configuration parameters are valid depends on the DRA driver, Kubernetes
> only passes them through without interpreting them.

Reply:
OK, SGTM. Thanks.